cmd/lint-panics/gopls/gopls.go

```diff
@@ -0,0 +1,153 @@
+package gopls
```

Comment:
@igor-sirotin, I'm under the impression the `gopls/` directory is mostly boilerplate and that the code is unrelated to the panic linter. Wouldn't it be better to move this whole package out of the `lint-panics/` directory? That way we facilitate the creation of new linters in the future by keeping the language server part decoupled a bit more.

Reply:
Yeah, it is quite some boilerplate, I was surprised that I had to write this. I'm still unsure that I did it right here, but didn't find any better way.

We can extract it, but same as for `parseFile`, I'd rather do that when it's needed, e.g. when we decided to write a new linter or smth.